packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingInvoiceDetails.tsx

```diff
@@ -293,25 +322,39 @@ const ExistingInvoiceDetailsBase = (props: ExistingReceivableDetailsProps) => {
               sx={{
                 display: 'flex',
                 flexDirection: 'column',
-                height: '100%',
+                // height: '100%',
```

Comment:
Probably, it would be better to remove it; or uncomment it?

Reply:
U're right, forgot to remove it. Thank you

Reply:
[cb8dce4](https://github.com/team-monite/monite-sdk/pull/71/commits/cb8dce4e1771b6cf5894bb5ab233ca84bd488359)